src/Client.php

```diff
@@ -42,7 +42,7 @@
  *
  * @template-implements \IteratorAggregate<string, static>
  */
-class Client implements ClientInterface, IteratorAggregate
+final class Client implements ClientInterface, IteratorAggregate
```

Comment:
This would be a breaking change, we can't do that.

Reply:
Oh nooooooooooooooooooooooo!

There is a `new static()` in it which will wrong if someone extends `Client`.
Reverting .... 🏃🏻‍♂️